content/docs/tutorials/get-started/experiments.md

```diff
@@ -1,173 +1,217 @@
+          python src/evaluate.py model.pkl \
+                 data/features scores.json prc.json
 ```
 
-</details>
-
-## Tuning parameters
-
-Let's say we want to try a modified feature extraction. The
-`src/featurization.py` script used to
-[create the pipeline](/doc/tutorials/get-started/data-pipelines#dependency-graphs-dags)
-actually accepts an optional third argument with the path to a YAML _parameters
-file_ to load values to tune its vectorization. Let's generate it:
+<details>
 
-```dvc
-$ echo "max_features: 6000" > params.yaml
-$ echo "ngram_range:" >> params.yaml
-$ echo "  lo: 1" >> params.yaml
-$ echo "  hi: 2" >> params.yaml
-$ git add params.yaml
+### 💡 Expand to see what happens under the hood
+
+It's pretty much the same as before, it generates a new stage in the `dvc.yaml`
+file:
+
+```yaml
+evaluate:
+  cmd: python src/evaluate.py model.pkl data/features scores.json prc.json
+  deps:
+    - data/features
+    - model.pkl
+    - src/evaluate.py
+  metrics:
+    - scores.json:
+        cache: false
+  plots:
+    - prc.json:
+        cache: false
 ```
 
-> Notice that we're versioning our parameters file with Git, in case we want to
-> change its contents for further experiments.
+The biggest difference is that we see two new sections `metrics` and `plots`
+that are used to mark certain files containing experiment "telemetry". Metric
+files contain scalars (e.g. `AUC`) and plots contain matrices and arrays of
+numbers that are supposed to be visualized to compare or make sense of them.
 
-Let's now redefine the featurization stage so that DVC knows that it depends on
-the specific values of `max_features` and `ngram_range`. For this we use the
-`-p` (`--params`) option of `dvc run`. `params.yaml` is the default parameters
-file name in DVC, so there's no need to specify this:
+`cache: false` means that those file are small enough and versioned directly
+with Git.
 
-```dvc
-$ dvc run -y -f featurize.dvc \
-          -d src/featurization.py -d data/prepared \
-          -p max_features,ngram_range.lo,ngram_range.hi \
-          -o data/features \
-          python src/featurization.py \
-                 data/prepared data/features params.yaml
+</details>
 
-$ git add featurize.dvc
-$ git commit -m "Update featurization stage"
+DVC expects `evaluate.py` to
+[write](https://github.com/iterative/example-get-started/blob/master/src/evaluate.py#L35)
+model
+[AUC](https://towardsdatascience.com/understanding-auc-roc-curve-68b2303cc9c5)
+value to the `scores.json` file which is marked as a metrics file with `-M`:
+
+```json
+{ "auc": 0.57313829 }
 ```
 
-> Please refer to `dvc params` for more information.
+And it
+[writes](https://github.com/iterative/example-get-started/blob/master/src/evaluate.py#L31)
+`precision`, `recall`, `thresholds` arrays it gets with the
+[`precision_recall_curve`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_curve.html)
+call into the `prc.json`:
```

Comment:
I'd avoid this [write] link. It's kind of repeated. Also simplify to:

`precision`, `recall`, `thresholds` arrays into `prc.json` (using [`precision_recall_curve`]):